operations/business-operations/self-service.md

```diff
@@ -2,40 +2,42 @@
 
 ![](../../.gitbook/assets/self-service.png)
 
-### Self Service will be enabled throught the combined capabilities of three ongoing projects
+### Self Service is enabled through the combined capabilities of three major components
 
 * Customer Web Portal
 * blapi \(Business Logic API\)
 * License Generator
 
-### To ask us questions and find out more [here](https://community.mattermost.com/private-core/channels/customer-portal)
+**Find the Self Service Roadmap in [ProductBoard](https://mattermost.productboard.com/roadmap)**
+
+**Ask questions [here](https://community.mattermost.com/private-core/channels/customer-portal)**
 
 ## Customer Web Portal
```

Comment:
Question about this: In the docs we refer to the "Customer Portal". More than happy to change it to match any changes I may have missed, but wanted to confirm what we're calling it before I do. :)

Reply:
Updated!